src/libstd/fs.rs

```diff
@@ -422,6 +422,11 @@ impl OpenOptions {
     /// This option, when true, means that writes will append to a file instead
     /// of overwriting previous contents.
     ///
+    /// Note that this isn't the same as `write` with the cursor starting at
+    /// the end of the file; this mode only allows writing to the end of the
```

Comment:
The part I don't understand is:

> Note that this isn't the same as `write` with the cursor starting at the end of the file

I would claim that `append` is the exact same as the cursor starting at the end of the file